matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.js

```diff
@@ -828,9 +841,7 @@ var PhenoTips = (function (PhenoTips) {
     _aggregatePhenotypes : function(match)
     {
         var matchPhenotypes = [];
-        var allPhenotypes = match.matched.phenotypes.predefined.concat(match.reference.phenotypes.predefined)
-                                                               .concat(match.reference.phenotypes.freeText)
```

Comment:
Why did we remove that `.concat` - because phenotypes is now a different structure?